wordpress-mobile/WordPress-iOS
# Blaze: Show blaze overlay with post content

Part of #20093 

## Description
- Shows a blaze overlay modal from posts list and pages list entry points
- Dismissed blaze overlay when blaze webview is dismissed

## Notes
- The following will be addressed in future PRs
  - UI refinement
  - Analytics 

## How to test

1. Switch to a Blaze eligible site
2. Go to the posts list screen
3. Tap on a Blaze eligible post
4. Tap on the ellipsis button, then tap "Promote with Blaze"
5. ✅ A Blaze overlay should be displayed with the post content
6. Tap on "Blaze this"
7. ✅ A Blaze webview opens and you land in the "Appearance" page
8. Close the webview
9. ✅ The overlay is also dismissed
10. Repeat the steps above for the pages list screen

No featured image | With featured image
--- | ---
<image src="https://user-images.githubusercontent.com/6711616/222725577-88c0c2d2-2bc9-4183-bd70-47135fcbdfc6.png" width=200> | <image src="https://user-images.githubusercontent.com/6711616/222725582-b53a1f98-8059-460a-8d6f-8bbbfde7915e.png" width=200>



## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
